lib/mix/tasks/rustler.download.ex

```diff
@@ -0,0 +1,57 @@
+defmodule Mix.Tasks.Rustler.Download do
+  @shortdoc "Download precompiled NIFs and build checksums"
+
+  @moduledoc """
+  This is responsible for downloading the precompiled NIFs for a given module.
```

Comment:
What happens if the files were already downloaded? Do we download them again? Do we ask users?

Reply:
We are downloading again. I think this is fine because the files are small. Do you think we should ask?

Reply:
It is fine as is, I would just mention it in the docs. :)

Reply:
I actually think forcing by default is the best way to go, otherwise you can endup with stale checksums.